lib/mongoidable/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Mongoidable
-  VERSION = "0.2.0"
+  VERSION = "0.2.1"
```

Comment:
Should this be a larger version bump given the dependency is a major upgrade?

Reply:
OK